src/components/Modal/Modal.test.tsx

```diff
@@ -83,13 +83,21 @@ describe("Modal ", () => {
     const closeButton = container.querySelector("button.p-modal__close");
     const cancelButton = container.querySelector("button#test-cancel");
 
-    expect(closeButton).toHaveFocus();
+    expect(cancelButton).toHaveFocus();
 
     await user.tab();
 
-    expect(cancelButton).toHaveFocus();
+    expect(closeButton).toHaveFocus();
+  });
 
-    await user.tab();
+  it("traps focus within the modal close button", async () => {
```

Comment:
The name of the test seems a bit misleading, maybe "focuses on close button if there are no other focusable elements"?

Reply:
changed as well.